unixfs.go

```diff
@@ -159,9 +159,9 @@ func size(pbdata *pb.Data) (uint64, error) {
 	switch pbdata.GetType() {
 	case pb.Data_Directory, pb.Data_HAMTShard:
 		return 0, errors.New("can't get data size of directory")
-	case pb.Data_File:
+	case pb.Data_File, pb.Data_Raw:
 		return pbdata.GetFilesize(), nil
-	case pb.Data_Symlink, pb.Data_Raw:
```

Comment:
On Raw types, data 0 bytes, but the filesize is set.

i.e.

`Type:Raw Data:"" filesize:114178 blocksizes:114178`